test/DynamoCoreTests/DynamoModelTestBase.cs

```diff
@@ -154,10 +154,10 @@ protected void OpenModelInManualMode(string relativeFilePath)
             CurrentDynamoModel.ExecuteCommand(new DynamoModel.OpenFileCommand(openPath, true));
         }
 
-        protected void OpenSampleModel(string relativeFilePath)
+        protected void OpenSampleModel(string relativeFilePath, bool forceManualExecution = false)
```

Comment:
I actually think this is a binary API break for clients who derive from this class for their tests, which is public, I think you need to add a new overload.